csrc/ir/utils.h

```diff
@@ -650,6 +650,8 @@ std::optional<std::vector<int64_t>> computePermutation(
 
 bool hasTrivialAllocationDomain(const TensorView* tv);
 
+bool hasUniformSiblings(Expr* expr);
```

Comment:
Comment was lost in the copy-paste
```suggestion
// Returns true if all expr outputs should be mapped unconditionally
bool hasUniformSiblings(Expr* expr);
```